### ISSUE TITLE
A click on "reload all" should only close the context menu, not the whole bookmark menu.

### ISSUE THREAD

**Issue by foobar13373** [nt1m/livemarks#360]
When right-clicking on a feed, hovering "Livemarks" and then clicking on the reload-all button, all Livemarks get reloaded, but the whole menu is closing.

In the past, when Firefox could still handle feeds by itself, the reload all button only closed the context menu, not the whole bookmark menu. Then you could directly see the new items. But with Livemarks you have to re-click through the whole menu again.

**Comment by foobar13373** [comment]
Any news here?

**Comment by evilpie** [comment]
Nothing we can do here.

**Comment by foobar13373** [comment]
Is this a limitiation of the extension API? Because Firefox itself has non-menu-closing context menus, like described in the issue.

**Comment by evilpie** [comment]
Yes there doesn't seem to be a way to specify that the bookmarks menu shouldn't be closed when clicking something on the context menu. We use this API: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/menus/create.
